docs/cimpl6ToolingReference.md

```diff
@@ -229,20 +229,21 @@ When specifying an Entry in the `target` array, use the [fully qualified name (F
 
 These configurations are used to control the production of the IG. The contents of the `implementationGuide` object are as follows:
 
-|Parameter                 |Type     |Description                                                          |
-|:-------------------------|:--------|:--------------------------------------------------------------------|
-|`npmName`                 |`string` |The assigned node package manager name for this IG, for example "hl7.fhir.us.mcode". The npm name is usually assigned by HL7.   |
+|Parameter                 |Type     |Description     |
+|--------------------|--------|-----------------|
+|`npmName`|`string` |The assigned node package manager name for this IG, for example "hl7.fhir.us.mcode". The npm name is usually assigned by HL7.   |
 |`version` |`string` |The version of this IG (not necessarily the version of FHIR), usually in the form _major.minor.revision_, for example, "3.0.1"  |
-|`ballotStatus`            |`string` |The HL7 ballot status of the IG (e.g., STU1 Ballot, Continuous Integration Build, etc.)      |
+|`ballotStatus`  |`string` |The HL7 ballot status of the IG (e.g., STU1 Ballot, Continuous Integration Build, etc.)      |
 |`packageList` |`string` |The name of the file to use as the [IG's package list](#package-list-file), relative to the Specification Directory. |
-|`includeLogicalModels`    |`boolean`|A "true" or "false" value indicating whether to include logical models in the IG.     |
-|`includeModelDoc`         |`boolean`|A "true" or "false" value indicating whether to include the model documentation in the IG. |
+|`includeLogicalModels`   |`boolean`|A "true" or "false" value indicating whether to include logical models in the IG.     |
```

Comment:
the words true and false should be italicized, not quoted

Reply:
fixed